Dockerfile

```diff
@@ -29,4 +29,6 @@ COPY requirements.txt /usr/src/app/
 RUN  pip install --no-cache-dir -r requirements.txt
 COPY . /usr/src/app
 
+ENV ES_URL "https://search-s4s-logs-xsjsafiwd7vkpiucmjqmdjkp7y.us-west-2.es.amazonaws.com/test-suite/log/"
```

Comment:
Can we leave this as a [deploy-time config setting](https://github.com/sync-for-science/reference-stack-docker/blob/master/deploys/demo.syncfor.science/docker-compose.yml#L48)? That will mean that our "prod" db only sees results from prod runs.